.vscode/c_cpp_properties.json

```diff
@@ -3,12 +3,11 @@
         {
             "name": "sm-wasi",
             "includePath": [
-                "${workspaceFolder}/runtime/spidermonkey/debug/include",
-                "${workspaceFolder}/runtime/spidermonkey/release/include",
+                "${workspaceFolder}/runtime/StarlingMonkey/include",
+                "${workspaceFolder}/runtime/StarlingMonkey/builtins/web",
+                "${workspaceFolder}/runtime/fastly/build-debug/",
                 "/opt/wasi-sdk/share/wasi-sysroot/include/",
-                "${workspaceFolder}/runtime/js-compute-runtime",
-                "${workspaceFolder}/runtime/js-compute-runtime/build/openssl-3.0.7/include",
-                "${workspaceFolder}/runtime/js-compute-runtime/third_party/fmt/include"
+                "${workspaceFolder}/runtime/fastly/host-api"
```

Comment:
I'm trying to work out how to get intellisense to configure properly, but am still not finding this working. Any suggestions very much welcome, but at least this is a step in the right direction.

Reply:
`cmake` can generate a `compile_commands.json`, which might be more reliable. Then you can configure `clangd` to read that instead.

Reply:
Amazing thank you, this worked.